### PR TITLE
Fix string checking bug and refreshLang param

### PR DESCRIPTION
**Pull request by samwilson**
The initLangSelect() method was treating the return value of
strstr() as an integer, which was resulting in a warning:
"A non-numeric value encountered at Intuition.php:1347"

This patch fixes this and makes it clearer what's happening.
It also fixes refreshLang(), which was previously not working.

### CODE REVIEW

**Comment by MusikAnimal** [comment]
:+1: I've observed the same warning

File: src/Intuition.php [hunk 1]
```diff
@@ -1360,8 +1360,10 @@ protected function initLangSelect( $option ) {
 					continue;
 				}
 
-				while ( ( $n = strstr( $acceptLang, '-' ) ) !== false ) {
```
Comment (Krinkle) [inline src/Intuition.php]: Oops, I guess this was previously intended to be `strpos()` - which is similar, but returns an integer indeed. Anyhow, this seemingly never worked right. Thanks!

File: src/Intuition.php [hunk 2]
```diff
@@ -1417,10 +1419,10 @@ public function listToText( $l ) {
 	 * Use this when you've changed the cookies and don't want to refresh
 	 * for it to be applied.
 	 *
-	 * @return true
+	 * @return bool Always true.
 	 */
 	public function refreshLang() {
-		$this->initLangSelect();
+		$this->initLangSelect( false );
```
Comment (Krinkle) [inline src/Intuition.php]: I recognise that it was confusing, given it had no default, and wasn't passed. However, just to confirm, this wasn't causing a bug right?

If so, I'd prefer to keep it omitted and instead make null the explicit default.

Comment (samwilson) [reply src/Intuition.php]: Yes, sure thing. It was giving an error, but fixing it with a default value works just as well.

**Comment by samwilson** [comment]
I guess that new lang code should be in a separate commit shouldn't it?

**Comment by samwilson** [comment]
I've made #93 for that.

**Comment by MusikAnimal** [comment]
Any chance we could get a new minor release that includes this patch?

**Comment by Krinkle** [comment]
@MusikAnimal This is now done (v1.2.0).

**Pull request merged**